addons/l10n_in_ewaybill_irn/models/l10n_in_ewaybill.py

```diff
@@ -44,7 +44,7 @@ def _prepare_ewaybill_transportation_json_payload(self):
 
     def _ewaybill_generate_irn_json(self):
```

Comment:
```suggestion
    def _ewaybill_generate_irn_json(self, raise_if_irn_not_found=True):
```
We don't want to break the compute at the same time :)